### DISCUSSION TITLE
استفاده از XRAY_FALLBACKS_INBOUND_TAG

### DISCUSSION THREAD

**Discussion by arian24b** [Gozargah/Marzban#163] [Q&A]
سلام
من از FALLBACKS_INBOUND استفاده کردم و زمانی که اجرامی کنم مرزبان رو با خطا مواجه میشه

چطور باید از XRAY_FALLBACKS_INBOUND_TAG استفاده کنم؟

**Comment by SaintShit** [comment]
سلام نمونش توی [fully-single-port](https://github.com/Gozargah/Marzban-examples/tree/master/fully-single-port) انجام شده. باید تگ اینباندی که شامل فالبک ها میشه رو توی فایل env تعیین کنید
`XRAY_FALLBACKS_INBOUND_TAG = FALLBACKS_INBOUND_TAG`

**Comment by omid-sadeghi** [comment]
این [ویدئوی یوتیوب](https://www.youtube.com/watch?v=CX8bXLD69G0&t=1638s) خوب و کامل توضیح داده همه چیز فالبک رو.
